<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>745</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>965</width>
    <height>745</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>965</width>
    <height>745</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QListWidget" name="noticeListWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>181</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="onlineListWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>181</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>公告</string>
   </property>
  </widget>
  <widget class="QLabel" name="onlineLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>当前在线人数</string>
   </property>
  </widget>
  <widget class="QListWidget" name="contentListWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>531</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputTextEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>540</y>
     <width>531</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendTextButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>620</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendImageButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
